### ISSUE TITLE
Update VSCODE documentation for xdebug 3

### ISSUE THREAD

**Issue by that0n3guy** [lando/lando#2732]
Please complete the following:

**Link to the page that needs to be updated or changed.**
https://docs.lando.dev/guides/lando-with-vscode.html

**Describe the change you'd like**
There is a bunch of stuff that needs updated on this page.  Here is a quick list:

* in lando.yml:   `xdebug: debug` (instead of `true`, thought i'm not sure if it makes a difference), also the config vs conf got me for a long time
* the default port change in `.vscode/launch.json` to 9003 instead of 9000
* in php.ini:

```
[PHP]

; Xdebug
xdebug.mode=debug ; it didn't work without this
xdebug.start_with_request = yes
xdebug.client_host = ${LANDO_HOST_IP}
xdebug.log = /tmp/xdebug.log
```

**Comment by milkovsky** [comment]
Thank you for the solution!
I love Open Source :D 
I've got this problem on Friday and the soluton is ready a few days after.

**Comment by paulostarling** [comment]
I can debug the app after I changed the port to 9003, but I'm not able to debug tests. Do you know what changes I need to do to debug tests? I have this problem after upgrade to v3.0.22...

**Comment by that0n3guy** [comment]
I've not debugged any tests w/ xdebug... but I'd start looking into xdebug 3's documentation and the update from 2 to 3 docs.  You'll probably find your info there.

**Comment by paulostarling** [comment]
Not sure if the problem is the port as it is not mentioned in the documentation. I didn't see nothing about the changes to be made to debug tests. I'll keep trying here...

**Comment by that0n3guy** [comment]
One thing I've noticed.  Sometimes xdebug stops working but a quick `lando rebuild -y` takes care of it instead.   I've started using `lando rebuild` instead of `lando start` when I open my projects now.

**Comment by paulostarling** [comment]
Yes! I've noticed this before too, but that is not the case I have now.

**Comment by thomas-j-moffett** [comment]
@musicaljoeker You sir, are a gentleman and a scholar. That message has been bugging me for a long time and only today it was just enough for me to search how to stop it. Thanks!
